starwars/src/Components/Card.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import './Card.css'
+import PropTypes from "prop-types";
+import 'bootstrap/dist/css/bootstrap.css';
+
+const CardImgs = (props) => {
+  const post = props.wars;
```

Comment:
Instead of doing this, you can use object destructuring:

```
const CardImgs = ({ wars }) => {

...

    <div>Name: <strong>{wars.name}</strong></div>
```